docs/style.css

```diff
@@ -85,11 +85,6 @@ img {
     display: inline-block;
 }
 
-.image-container {
-    display: block;
-    margin: 20px 0;
```

Comment:
Good catch 😅 .

Would you mind migrating this `margin` line to the retained instance of `.image-container`?